reel-picker/src/main/java/com/innowisegroup/reelpicker/datetime/LocalTime.kt

```diff
@@ -15,13 +15,35 @@ class LocalTime private constructor(val hour: Int, val minute: Int) : Serializab
 
     override fun hashCode(): Int = 31 * hour + minute
 
+    fun plusHours(hoursToAdd: Int): LocalTime =
+        if (hoursToAdd == 0) {
+            this
+        } else {
+            val newHour = hour + hoursToAdd
+            of(newHour, minute)
+        }
```

Comment:
Tests didn't pass with such implementation in case when user tries to add more than 24 hours

Reply:
![image](https://user-images.githubusercontent.com/79689735/186514389-d45a9565-b405-43fe-9276-6ad464f9fd40.png)